app/models/plant_trial.rb

```diff
@@ -8,6 +8,37 @@ class PlantTrial < ActiveRecord::Base
   include Filterable
   include Pluckable
 
+  validates :plant_trial_name,
+            presence: true
+
+  validates :project_descriptor,
+            presence: true
+
+  validates :plant_trial_description,
+            presence: true
+
+  validates :trial_year,
+            presence: true,
+            length: { is: 4 }
+
+  validates :institute_id,
+            presence: true
+
+  validates :trial_location_site_name,
+            presence: true
+
+  validates :place_name,
+            presence: true
+
+  validates :latitude,
+            presence: true
```

Comment:
Why, though? Existing records seem to have proper values in this field. Granted, it's not a GIS datatype, but Cropstore is not a GIS system so perhaps we don't need to roll out that particular cannon.